### DISCUSSION TITLE
What's the point of `env` in next.config.js instead of a direct import?

### DISCUSSION THREAD

**Discussion by kud** [vercel/next.js#21716] [Help] [answered]
Hello,

I was wondering what's the point of using `env`  in `next.config.js` via `process.env` instead of using something like

```js
// {root}/media-queries.js

const MQ = {
  "--tablet-and-below": "(max-width: 1279px)",
  "--desktop": "(min-width: 1280px)",
}

export default MQ
```

```js
// anyFile.js

import MQ from "media-queries"

console.log(MQ["--tablet-and-below"]) // (max-width: 1279px)
```

?

**Comment by jamesmosier** [comment, marked as answer]
Next's [environment variable support](https://nextjs.org/docs/basic-features/environment-variables) is best used for application secrets that may change between environments (i.e. production vs development) or any other secrets that you wanted to manage with your deployments. 

For something like media queries what you have (in a separate JS file) would work best in my opinion!

**Reply by kud** [reply]
I'm sorry, I didn't read enough the doc in fact.

I know what are envs in fact, but I didn't read this:

```
-  Use .env.local to load environment variables
-  Expose environment variables to the browser
```

which now makes totally sense to me. 😊

Thanks for your time @jamesmosier
